bioblend/galaxy/workflows/__init__.py

```diff
@@ -205,15 +212,23 @@ def export_workflow_dict(self, workflow_id: str, version: Optional[int] = None)
         :type version: int
         :param version: Workflow version to export
 
+        :type style
+        :param style: Set style to `format2` to get gxformat2 encoded workflow
```

Comment:
Maybe explain a bit both formats:

```suggestion
        :param style: Either "ga" for the original JSON format or "format2" for
          the modern YAML gxformat2 format.
```